clang/lib/3C/AVarBoundsInfo.cpp

```diff
@@ -674,15 +747,18 @@ ABounds *AVarBoundsInfo::getBounds(BoundsKey L, BoundsPriority ReqP,
   return nullptr;
 }
 
-bool AVarBoundsInfo::updatePotentialCountBounds(BoundsKey BK,
-                                                std::set<BoundsKey> &CntBK) {
-  bool RetVal = false;
-  if (!CntBK.empty()) {
-    auto &TmpK = PotentialCntBounds[BK];
-    TmpK.insert(CntBK.begin(), CntBK.end());
-    RetVal = true;
-  }
-  return RetVal;
+bool
+AVarBoundsInfo::updatePotentialCountBounds(BoundsKey BK,
```

Comment:
These two methods always return `true` now. Is it correct for them not to handle the `CntBK.empty()` case? Assuming it is fine, can we make them `void`?

Reply:
Done.